web/package.json

```diff
@@ -35,6 +35,7 @@
         "bootstrap": "npm install && lerna bootstrap --hoist",
         "build": "lerna run build --stream",
         "build:debug": "cross-env NODE_ENV=development CARGO_FEATURES=avm_debug npm run build",
+        "build:full-release": "cross-env RUFFLE_REAL_FANCY_WASM=true npm run build",
```

Comment:
Now that it is more internal, because users can use the `build:full-release` script instead of setting this directly, perhaps this would be a better name indeed.